src/sentry/static/sentry/app/components/onboardingWizard/task.tsx

```diff
@@ -81,26 +81,28 @@ function Task({router, task, onSkip, onMarkComplete, forwardedRef, organization}
     const completedOn = moment(task.dateCompleted);
 
     return (
-      <ItemComplete ref={forwardedRef} onClick={handleClick}>
-        <StatusIndicator>
-          {task.status === 'complete' && <CompleteIndicator />}
-          {task.status === 'skipped' && <SkippedIndicator />}
-        </StatusIndicator>
-        {task.title}
-        <CompletedDate title={completedOn.toString()}>
-          {completedOn.fromNow()}
-        </CompletedDate>
-        {task.user ? (
-          <TaskUserAvatar hasTooltip user={task.user} />
-        ) : (
-          <Tooltip
-            containerDisplayMode="inherit"
-            title={t('No user was associated with completing this task')}
-          >
-            <TaskBlankAvatar round />
-          </Tooltip>
-        )}
-      </ItemComplete>
+      <TaskCard ref={forwardedRef} onClick={handleClick}>
+        <CompleteTitle>
+          <StatusIndicator>
+            {task.status === 'complete' && <CompleteIndicator />}
```

Comment:
maybe we should destructure these props?

Reply:
I can try to do this in another PR 👍